content/blog/2016/2016-03-02-gsoc2016-announcement.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Jenkins participates in Google Summer of Code 2016"
```

Comment:
I can't quite explain it but this reads funny to me. How about "Jenkins joins in the Google Summer of Code 2016"?